content/references/http-websocket-apis/public-api-methods/server-info-methods/nft_info.md

```diff
@@ -0,0 +1,168 @@
+      "This server may be out of date"
+    ]
+  }
+```
+
+*JSON-RPC*
+
+```json
+{
+  "result": {
+    "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C",
+    "ledger_index": 2436210,
+    "owner": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "is_burned": false,
+    "flags": 0,
+    "transfer_fee": 0,
+    "issuer": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "nft_taxon": 0,
+    "nft_sequence": 12,
+    "uri": "ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf4dfuylqabf3oclgtqy55fbzdi",
+    "validated": true,
+    "status": "success",
+    "warnings": [
+      "This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request",
+      "This server may be out of date"
+    ]
+  }
+}
+```
+
+*Commandline*
+
+```json
+{
+  "result": {
+    "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C",
+    "ledger_index": 2436210,
+    "owner": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "is_burned": false,
+    "flags": 0,
+    "transfer_fee": 0,
+    "issuer": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "nft_taxon": 0,
+    "nft_sequence": 12,
+    "uri": "ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf4dfuylqabf3oclgtqy55fbzdi",
+    "validated": true,
+    "status": "success",
+    "warnings": [
+      "This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request",
+      "This server may be out of date"
+    ]
+  }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format][], with a successful result containing an `info` object as its only field.
+
+The `info` object may have some arrangement of the following fields:
+
+| `Field`                             | Type            | Description          |
+|:------------------------------------|:----------------|:---------------------|
+| `nft_id`                            | String          | A unique identifier for the non-fungible token (NFT). |
+| `ledger_index`                      | Integer         | _(Omitted if `ledger_current_index` is provided instead)_ The [ledger index][] of the most recent ledger version where this NFT's state was changed. For example, when the NFT was minted, its ownership changed, or it was burned. The information returned contains whatever happened most recently compared to the input ledger index. |
+| `owner`                             | String          | The account ID which denotes the owner of this NFT at the above ledger index. |
+| `is_burned`                         | Boolean         | When `true` i.e., NFT is burned at this ledger, this field returns the URI of the NFT if available or NULL if the URI is unavailable. Returns `false` if the NFT is not burned at this ledger. |
+| `flags `                            | Integer         | The flag set of this NFT. |
+| `transfer_fee`                      | Integer         | The transfer fee of this NFT. |
```

Comment:
We should also add some clarity on how transfer fees are represented. See the existing documentation for `NFTokenMint` transaction